Src/Web/Web.Shared.Net.Tests/Helpers/HttpModuleHelper.cs

```diff
@@ -67,7 +67,9 @@ public static HttpContext GetFakeHttpContext(IDictionary<string, string> headers
 
             var workerRequest = new SimpleWorkerRequestWithHeaders(UrlPath, UrlQueryString, new StringWriter(CultureInfo.InvariantCulture), headers);
             
-            return new HttpContext(workerRequest);
+            var context = new HttpContext(workerRequest);
+            HttpContext.Current = context;
```

Comment:
Every MSTest test is executed in it's own AppDomain, it does not affect other tests